.circleci/config.yml

```diff
@@ -1099,6 +1099,10 @@ workflows:
             - b_bytecode_osx
             - b_bytecode_ems
 
+      # Code Coverage enabled build and tests
+      - b_ubu_codecov: *workflow_trigger_on_tags
+      - t_ubu_codecov: *workflow_ubuntu2004_codecov
```

Comment:
`t_ubu_codecov` job is running with `EVM: constantinople`. Why not `istanbul`? Actually, lots of jobs, including `t_osx_soltest` are hard-coded for `constantinople`. What's up with that?